test/__manifest__.py

```diff
@@ -6,8 +6,21 @@
     'summary': 'Test Module',
     'website': 'trescloud.com',
     'depends': [
+        'sale_management',
+        'account',
+        'stock'
     ],
     'data': [
+        # data
+        # security
+        'security/ir.model.access.csv',
+        'security/delivery_detail_security.xml',
+        # reports
+        'reports/report_invoice_templates.xml',
+        # wizard
+        # views
+        'views/account_move_views.xml',
+        'views/delivery_detail_views.xml',
     ],
     'installable': True,
     'application': True,
```

Comment:
```suggestion
    'application': False,
```

No es una aplicación